catalog/nsili/catalog-nsili-transformer/src/main/java/com/connexta/alliance/nsili/transformer/DAGConverter.java

```diff
@@ -1143,29 +1143,23 @@ private static String collectionToString(Collection<String> collection) {
 
     public static void logMetacard(Metacard metacard, String id) {
         MetacardType metacardType = metacard.getMetacardType();
-        LOGGER.trace("{} : ID : {}", id, metacard.getId());
-        LOGGER.trace("{} :  Metacard Type : {}",
-                id,
-                metacardType.getClass()
-                        .getCanonicalName());
-        LOGGER.trace("{} :  Title : {}", id, metacard.getTitle());
+        LOGGER.trace("{} : ID : " + metacard.getId(), id);
+        LOGGER.trace("{} :  Metacard Type : " + metacardType.getClass()
+                .getCanonicalName(), id);
+        LOGGER.trace("{} :  Title : " + metacard.getTitle(), id);
         if (metacard instanceof MetacardImpl) {
-            LOGGER.trace("{} :  Description : {}", id, ((MetacardImpl) metacard).getDescription());
+            LOGGER.trace("{} :  Description : " + ((MetacardImpl) metacard).getDescription(), id);
```

Comment:
That change is part of the CAL-14 / CAL-17 PR. One of them will need to be rebased depending on what gets merged first.